### ISSUE TITLE
Python 2.7.10: module webbrowser missing

### ISSUE THREAD

**Issue by bouwew** [Entware-ng/Entware-ng#58]
I have installed Entware-ng on my QNAP TS-419P NAS, to be able to run Sickrage and Sabnzbd.
I have installed the Entware-ng Python version, unfortunately I get the following error when starting Sickrage:

Starting SickBeard-TVRage
Traceback (most recent call last):
  File "SickBeard.py", line 66, in <module>
    import sickbeard
  File "/share/MD0_DATA/.qpkg/SickBeard-TVRage/SickBeard-TVRage/sickbeard/__init__.py", line 19, in <module>
    import webbrowser
ImportError: No module named webbrowser

I have tried to google the problem, most obvious solution is to install the module, however the command 'pip install webbrowser' gives this:

Collecting webbrowser
  Could not find a version that satisfies the requirement webbrowser (from versions: )
No matching distribution found for webbrowser

Apparently there is not a separate downloadable module for it.

Any help you can provide?

**Comment by zyxmon** [comment]
>> on my QNAP TS-419P NAS

How I can guess what cpu/feed you are using? Let us suppose it is armv5.

>> No matching distribution found for webbrowser

**Comment by bouwew** [comment]
Thank you for you quick response!!

I will test tomorrow, will let you know the result.

**Comment by bouwew** [comment]
I have installed the provide ipk-file and I have recompiled everything successfully.
After that I could successfully start Sabnzbd and Sickrage!
Thanks! Your great support is very much appreciated!
I would like to suggest that you make this updated ipk-file available for others. I will write a post on the QNAP-forum that I am successfully using Entware-ng with a fully working Python, if you don't mind.

You were correct, TS-419P is indeed armv5.
On your remark:"(line 199). May be there are problems with it." I'm quite familiar with Openwrt, I have been using it for years on my wireless routers. These routers are much less powerful than any NAS, so I can imagine that Openwrt devs don't want to enable the module webbrowser. This would load the processor of a router too much.

**Comment by zyxmon** [comment]
>> so I can imagine that Openwrt devs don't want to enable the module webbrowser. 
May be they are just saving size. AFAIR there was no python bz2 for some reason. It was disabled the same way as webbrowser. Now bz2 is in python ipk.

I'll enable webbrowser for python and update repo today. Thanks for testing.

**Comment by alveola** [comment]
The package [Autosub ](https://github.com/BenjV/autosub-bootstrapbill) also needs python and I have the same issue at this moment. I'm running QNAP TS-251+ (4.3.3.0154 firmware, so x86_64).
I succeeded in compiling a new version (2.7.13-5), but I can't install this. It seems that @bouwew was able to install an older package. I tried many command line options, but always get the message that the latest version is already installed:
> # opkg install python-light_2.7.13-5_x86_64.ipk
> Package python-light (2.7.13-4) installed in root is up to date.

What can I do to install this? Or is there an alternative solution for the missing webbrowser in python?
